config/initializers/ahoy.rb

```diff
@@ -1,3 +1,7 @@
-class Ahoy::Store < Ahoy::Stores::ActiveRecordTokenStore
-  Ahoy.track_visits_immediately = true
+class Ahoy::Store < Ahoy::DatabaseStore
+  def visit_model
+    Visit
+  end
 end
+
+Ahoy.user_agent_parser = :legacy
```

Comment:
This is a big difference; let's make sure it works.  (I don't think we have any tests for Ahoy, but we might.)

Reply:
@saracarl I added this to restore the original Ahoy behavior, some of our tests (`spec/features/ahoy_spec.rb`) break if we remove this option.